api/README.md

```diff
@@ -4,21 +4,17 @@
 
 In the project directory, you can run:
 
-### `npm install`
+### `yarn install`
```

Comment:
To be consistent with the readme file we can use `yarn` instead of `yarn install`

Reply:
I don't understand.  Everything is now consistent across the entry Readme and the API and client readme.  @lauriejim